packages/block-library/src/navigation-link/link-ui.js

```diff
@@ -146,6 +162,12 @@ export function LinkUI( props ) {
 		};
 	}
 
+	const link = {
```

Comment:
There are a lot of instances of props.link.whatever here. It might be worth destructuring that variable first so that we can just reference the variables directly. This will mean if we ever want to change the props we pass again we'll have less places to change. Happy to do that in a followup though.